lib/autocomplete.ts

```diff
@@ -61,9 +53,26 @@ export function autocomplete<T>(
     ...options
   };
 
-  const contentFunc = options.contentFunc!;
+  // Keeps track of the last value as typed by the user.
+  let lastAsTyped = inputElem.value;
+  const lis = dom.onElem(inputElem, 'input', () => lastAsTyped = inputElem.value);
```

Comment:
Not sure our tslint settings in this project, but elsewhere we always use curly braces for such functions, like `() => { lastAsTyped = inputElem.value; }`. I think it's clearer (when braces are omitted, it's equivalent to `() => { return expr; }`, which is more confusing in this case)